packages/insomnia/src/ui/components/websockets/realtime-response-pane.tsx

```
@@ -314,7 +314,7 @@ const RealtimeActiveResponsePane: FC<RealtimeActiveResponsePaneProps & { readySt
         <TabPanel className="flex w-full flex-1 flex-col overflow-hidden" id="events">
           <PanelGroup direction="vertical" className="grid h-full w-full grid-rows-[repeat(auto-fit,minmax(0,1fr))]">
             {response.error && !isMCPAuthError ? (
-              <ResponseErrorViewer url={response.url} error={response.error} />
+              <ResponseErrorViewer url={response.url} error={response.error} isMcpResponse />
```

Why is it hard-coded to `isMcpResponse`? 🤔

---

Refine the logic:
- For none MCP Auth related error, using `isMcpResponse` to decide whether to show settings or MCP certificate modal to `Disable SSL Validation`.
- For MCP Auth error, since users can view error details when clicking the event. Adding a new `showErrorDetails` to hide error details under this situation to avoid error details to cover events table by default